file-formats/dobj/dobj-v1.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/dobj/dobj-v1.json",
+  "title": "Data Destruction Object",
+  "description": "ILM destruction object (DOBJ) v1",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "ABAP File Format Version",
+      "description": "The ABAP file format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "description": {
+      "title": "Description",
+      "description": "Description",
+      "type": "string",
+      "maxLength": 60
+    },
+    "dobjDetails": {
+      "title": "Data destruction object",
+      "description": "Data destruction object",
+      "type": "object",
+      "properties": {
+        "destrProgram": {
+          "title": "Destruction Program",
+          "description": "Destruction Program",
+          "type": "string",
+          "maxLength": 60
+        },
+        "applicComponent": {
+          "title": "Application Component",
+          "description": "Application Component",
+          "type": "string",
+          "maxLength": 30
+        },
+        "structureDefinition": {
```

Comment:
regarding the fields... I have the impression that this is actually a parent-child relation with multiple levels. means a tree. this table is a flat list of that tree. 
a fully flexible multi-level parent-child is not possible (yet).

but you could at least imagine a list of one level parent child relation. maybe better for UX.

means:
parent table: table1
 |-dependent table: table1_1
 |-dependent table: table1_2
parent table: table1_2
 |-dependent table: table1_2_a

maybe with nice value help on parent table